Tasks/NuGetInstaller/VsoNuGetHelper.ps1

```diff
@@ -121,6 +121,19 @@ function SetCredentialsNuGetConfigAndSaveTemp
         [uri]$targetFeed
     )
 
+    #check that the config has a configuration section
+    if($nugetConfig.configuration -eq $null)
+    {
+        if($nugetConfig.packages -ne $null)
+        {
+            throw (Get-LocalizedString -Key "An invalid NuGet.config was passed to the NuGet Installer step. Confirm you did not add packages.config where NuGet.Config was expected")
```

Comment:
I'm no expert in wording... but, I feel like there might be a better & smaller message here : )